index.js

```diff
@@ -0,0 +1,7 @@
+const redis = require('redis');
```

Comment:
Is this file for test purposes? We could write an integration test for the redis plugin by extending codeship's codeship-services.yml:

here is an example from https://docs.cloudbees.com/docs/cloudbees-codeship/latest/pro-builds-and-configuration/services on how to run a sidecart postgres, the same way we could side-load redis.
```
app:
  build:
    image: codeship/app
    dockerfile: Dockerfile
  links:
    - postgres
postgres:
  image: healthcheck/postgres:alpine
```

Reply:
@Mikearaya ping

Reply:
@pozylon noted

Reply:
it was for test